guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -229,7 +229,7 @@
     "\n",
```

Comment:
<p>Open a terminal application, navigate to the directory you want to work in, and activate the conda&nbsp;<code>environment</code>&nbsp;you want to use with the ArcGIS API for Python. Install the API with the following command:</p><p> </p><p><a href="https://docs.pipenv.org/" target="_blank">Pipenv</a>&nbsp;is the official packaging tool for managing environments and installing packages from the&nbsp;<a href="https://pypi.python.org/pypi" target="_blank">Python Package Index (PyPI)</a>. To install the ArcGIS API for Python from PyPI in a new environment, create a new folder named&nbsp;<code>your-folder</code>. Then, open a terminal and run&nbsp;<code>cd /path/to/your-folder</code>&nbsp;to change directories into&nbsp;<code>your-folder</code>. Next, enter the following command to simultaneously create a new environment and install the API in it:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>